tox.ini

```diff
@@ -1,22 +1,29 @@
 [tox]
+isolated_build = True
 envlist = py36, linting
 
 [testenv]
-passenv = TOXENV CI TRAVIS TRAVIS_* PYBIND_PATH  APPVEYOR APPVEYOR_*
+whitelist_externals = poetry
+passenv = TOXENV CODECOV_TOKEN
 setenv =
     PYTHONPATH = {toxinidir}
-deps =
-    -r{toxinidir}/requirements_dev.txt
-    codecov
-    git+https://git@github.com/ESSS/hookman.git
 
-commands =
-    pytest --basetemp={envtmpdir} --cov=alfasim_sdk
-    codecov
+deps=
+    poetry
 
+commands =
+    poetry install -v
+    poetry run pytest --basetemp={envtmpdir} --cov=alfasim_sdk --cov-report=xml
+    poetry run codecov -t CODECOV_TOKEN
 
 [testenv:linting]
 skip_install = True
 basepython = python3.6
 deps = pre-commit>=1.11.0
 commands = pre-commit run --all-files --show-diff-on-failure
+
+# [tox:.package]
+# # note tox will use the same python version as under what tox is installed to package
```

Comment:
Is this required?

Reply:
I think that it's not necessary, even though the tox documentation recommends it (perhaps I didn't understand the docs correctly)

[Link for the section on docs documentation that mentions poetry](https://tox.readthedocs.io/en/latest/example/package.html#poetry)

Reply:
It is not really needed, we only support python3 anyway... please remove it. 👍